.github/workflows/lib-check.yml

```
@@ -72,7 +72,7 @@ jobs:
           path: .turbo
           key: ${{ runner.os }}-turbo-build:type-${{ github.sha }}
       - name: Check Formatting
-        run: yarn lint --cache-dir=".turbo"
+        run: yarn check:all
```

Don't we still need to tell yarn the cache dir?

```suggestion
        run: yarn check:all --cache-dir=".turbo"
```

---

check:all is not a turbo task

---

Oh my bad, got it.